e2e-test/src/util.js

```diff
@@ -1,2 +1,7 @@
-/* eslint-disable import/prefer-default-export */
 export const { CLIENT_URL } = process.env;
+
+export async function checkNoFile() {
+  // check if there are any other files hanging around
+  const uploadFileList = await expect(page).toClickXPath("//div[@class='v-list upload-list theme--light']");
```

Comment:
This doesn't seem to do what it advertises that it does. All it does is click on the upload list, which isn't even meaningful; I think puppeteer would just click in the middle of the upload list element, which will be somewhere on some element in the list, if it exists.
If you're checking that no files exist, you should return a boolean.